src/NzbDrone.Core/Indexers/Definitions/IPTorrents.cs

```diff
@@ -194,6 +194,8 @@ private IEnumerable<IndexerRequest> GetPagedRequests(string term, int[] categori
                 qc.Add(cat, string.Empty);
             }
 
+            qc.Add("p", ((offset / limit) + 1).ToString());
```

Comment:
What's the page size for IPTorrents? This has the possibility to result in odd paging if we don't also factor page size into the equation, especially if page size is variable.

Reply:
IPT defaults to 50 results per page, users can increase it up to 255.

With this change users should receive the same or more results regardless of their IPT max results setting, at least with Sonarr. Users with their IPT max set to < 100 will still receive at most a single page of results, since Sonarr assumes there are no further pages when less results are returned than the 100 it asked for.

Users with their max set to 100 or greater will receive the same first page of results as before, plus additional pages rather than page 1 over and over.

It's not ideal and there is the possibility of mismatches as further pages are parsed. How do you handle this for other indexers? I'm new to the arrs but some quick research indicated it was basically on the user to set the site setting to 100 results for the best experience. It was suggested to Jackett that they [warn the user](https://github.com/Jackett/Jackett/issues/12853) about it however it doesn't look like that went ahead.

I suppose it could also be handled by Prowlarr fetching and parsing multiple results pages as necessary to ensure it returns the proper number of results to each Sonarr request though that seems a fair bit more complicated.